drivers/net/macsec.c

```diff
@@ -933,6 +933,9 @@ static enum rx_handler_result handle_not_macsec(struct sk_buff *skb)
 		 * SecTAG, so we have to deduce which port to deliver to.
 		 */
 		if (macsec_get_ops(macsec, NULL) && netif_running(ndev)) {
+			if (hdr->h_proto == htons(ETH_P_PAE))
+				continue;
```

Comment:
Why not just consume it?
```
kfree_skb(skb);
return RX_HANDLER_CONSUMED;
```

Reply:
The idea is to keep such packets only in real_dev, not to drop them.

Reply:
Hmm, okay. For some reason I vaguely remember discussion to drop them...